src/sound.cpp

```diff
@@ -22,6 +22,8 @@
 #include "safeguards.h"
 
 static SoundEntry _original_sounds[ORIGINAL_SAMPLE_COUNT];
+/** The sound file for the original sounds, i.e. those not defined/overridden by a NewGRF. */
+static RandomAccessFile *_original_sound_file;
```

Comment:
Does this still need to be global? It doesn't appear to be used outside the OpenBankFile function

Reply:
And if not, it seems like a good candidate for using std::unique_ptr

Reply:
It populates the _original_sound entries that contain a pointer to it, and that pointer is used every time a sound is to be played. However, once we are done with it we must be able to free it, so it needs to be global in some manner. Or do you mean moving the static into OpenBankFile?

Reply:
Accidentally deleted this comment, see the next one down :)